test/Driver/print_target_info.swift

```diff
@@ -6,7 +6,7 @@
 
 // RUN: %swift_driver -print-target-info -target x86_64-unknown-linux -static-executable | %FileCheck -check-prefix CHECK-LINUX-STATIC %s
 // RUN: %swift_driver -print-target-info -target x86_64-unknown-linux -static-stdlib | %FileCheck -check-prefix CHECK-LINUX-STATIC %s
-// RUN: %target-swift-frontend -print-target-info -target x86_64-unknown-linux -use-static-resource-dir | %FileCheck -check-prefix CHECK-LINUX-STATIC %s
+// RUN: %swift_frontend_plain -print-target-info -target x86_64-unknown-linux -use-static-resource-dir | %FileCheck -check-prefix CHECK-LINUX-STATIC %s
```

Comment:
This fails because `%target-swift-frontend` now includes a `-resource-dir` flag set to the Swift shared libraries, which overrides this `-use-static-resource-dir` flag that's looking for the static resource directory. Since this command sets an alternate target anyway, the Android target flags are irrelevant and `%swift_frontend_plain` is enough to test this.

@drexin, you added this test, let me know what you think when you're back from break.

Reply:
Sounds good to me